sql/src/main/java/io/crate/plugin/SQLPlugin.java

```diff
@@ -190,5 +192,6 @@ public void onModule(ScriptModule scriptModule) {
     public void onModule(ActionModule actionModule) {
         actionModule.registerAction(SQLAction.INSTANCE, TransportSQLAction.class);
         actionModule.registerAction(SQLBulkAction.INSTANCE, TransportSQLBulkAction.class);
+        actionModule.registerAction(BulkCreateIndicesAction.INSTANCE, TransportBulkCreateIndicesAction.class);
```

Comment:
why do you need to register the action? There is no need to make this available for the client?